en/user_guide/websites_creation.rst

```diff
@@ -0,0 +1,93 @@
+Website creation
+================
+
+Open Orchestra is multisite and allows to create several websites from the same Back Office application.
+
+Name
+----
+
+The field name contributes the name of the website and he allows to generate the identifier of the website.
+
+L'identifiant du site
+---------------------
+
+The site id of field is generated when the name of the site is filled, but it remains alterable during the creation of the site.
```

Comment:
The site id can be generated from the site name or directly by the user at the creation.
In the remaining process, this field is not anymore editable